integration-cli/docker_cli_build_unix_test.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 func (s *DockerSuite) TestBuildResourceConstraintsAreUsed(c *check.C) {
-	testRequires(c, cpuCfsQuota)
+	// E2E: Test assumes no other containers running.
+	testRequires(c, cpuCfsQuota, NotE2E)
```

Comment:
I think we should fix this test instead of skipping it. There's no reason a build test should need a clean environment.

I think this can be fixed by adding a `--label` to the build command and then filtering by that label, but there may be an easier way.